src/Entity/User.php

```diff
@@ -499,4 +502,16 @@ public function setIpAddress(string $ipAddress): static
 
         return $this;
     }
+
+    public function isImperial(): ?bool
```

Comment:
Remove ?, it should not allow null values

Reply:
Done